animations/lib/main.dart

```diff
@@ -6,6 +6,7 @@ import 'dart:io';
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
+// ignore: depend_on_referenced_packages
```

Comment:
Ideally, we wouldn't need to continue using this override; but I've never encountered this lint before and am not sure what to do for plugins that come directly from Flutter

Reply:
Which package is failing `depend_on_referenced_packages`? By my reading, it's `package:window_size` which definitely should be directly referenced in `pubspec.yaml`, or am I missing something?

Reply:
And it is referenced. I'm confused as to why this lint is firing here. @bwilkerson halp?

Reply:
Seems to be a potential bug around git-based dependencies?